packages/ts-config-webpack-plugin/test/TsConfigWebpackPlugin.test.js

```diff
@@ -180,7 +180,7 @@ describe('TsConfigWebpackPlugin inside webpack context', () => {
 		compiler.run((err, stats) => {
 			console.warn = origWarn;
 			expect(Boolean(warnMessage)).toEqual(true);
-			const rootPath = path.resolve(__dirname, '..');
+			const rootPath = path.resolve(__dirname, '..').replace(/\\/g, '/');
```

Comment:
what about `path.posix.resolve(__dirname, '..');`?

https://nodejs.org/api/path.html#path_path_posix

Reply:
I've seen it but I am not aware of what it does and the proposed solution works and is used in more projects.

And other utility libs do it too.

Also see https://github.com/nodejs/node/issues/12298 and other related issues.

Reply:
You are right posix is not working as expected on windows:

<img width="574" alt="win7_posix" src="https://user-images.githubusercontent.com/4113649/52912582-83021f80-32b3-11e9-90e8-a1fcb31b4639.png">

Guess thats fine than 👍